build.gradle

```diff
@@ -1,9 +1,10 @@
 // Top-level build file where you can add configuration options common to all sub-projects/modules.
 buildscript {
-    ext.kotlin_version = "1.4.10"
+    ext.kotlin_version = '1.4.10'
     repositories {
         google()
         jcenter()
+        maven { url "https://jitpack.io" }
```

Comment:
Do we still need jitpack? Our SDK is uploaded to `mavenCentral`

Reply:
We have a dependency that comes from there, I'm working on getting rid of it 😄